[Bug] `dnd5e.preConfigureInitiative`

```js
Error: Error thrown in hooked function '' for hook 'dnd5e.renderChatMessage'. (intermediate value) is not iterable
[Detected 2 packages: automated-conditions-5e(13.512.1.1), system:dnd5e(5.1.2)]
    at Hooks.onError (foundry.mjs:23813:24)
    at 🎁Hooks.onError#0 (libWrapper-wrapper.js:188:54)
    at #call (foundry.mjs:23795:36)
    at Hooks.callAll (foundry.mjs:23750:17)
    at ChatMessage5e.renderHTML (chat-message.mjs:144:11)
    at async #postOne (foundry.mjs:103317:18)
    at async #try (foundry.mjs:7154:17)Caused by: TypeError: (intermediate value) is not iterable
[Detected 2 packages: automated-conditions-5e(13.512.1.1), system:dnd5e(5.1.2)]
    at _getTooltip (ac5e-helpers.mjs:1:1)
```

--- a/scripts/ac5e-hooks.mjs
+++ b/scripts/ac5e-hooks.mjs
@@ -708,8 +708,8 @@ export function _preConfigureInitiative(subject, rollConfig) {
 
 	ac5eConfig.advantageMode = advantageMode;
 	ac5eConfig.defaultButton = advantageMode === 0 ? 'normal' : advantageMode > 0 ? 'advantage' : 'disadvantage';
-
-	const ac5eConfigObject = { [Constants.MODULE_ID]: ac5eConfig, classes: ['ac5e'] };
+	_getTooltip(ac5eConfig);
+	const ac5eConfigObject = { [Constants.MODULE_ID]: ac5eConfig };
 	foundry.utils.mergeObject(rollConfig.options, ac5eConfigObject);
 	if (settings.debug) console.warn('AC5E._preConfigureInitiative', { ac5eConfig });
 	return ac5eConfig;

